src/main/java/org/broadinstitute/consent/http/service/ResearcherService.java

```diff
@@ -115,7 +120,7 @@ public void deleteResearcherSpecificProperties(List<UserProperty> properties) {
         rpForDAR.put(UserFields.DIVISION.getValue(), properties.getOrDefault(UserFields.DIVISION.getValue(), null));
         rpForDAR.put(UserFields.ERA_COMMONS_ID.getValue(), properties.getOrDefault(UserFields.ERA_COMMONS_ID.getValue(), null));
         rpForDAR.put(UserFields.PUBMED_ID.getValue(), properties.getOrDefault(UserFields.PUBMED_ID.getValue(), null));
-        rpForDAR.put(UserFields.PROFILE_NAME.getValue(), properties.getOrDefault(UserFields.PROFILE_NAME.getValue(), null));
+        rpForDAR.put(DarConstants.PROFILE_NAME, user.getDisplayName());
```

Comment:
It is common to find duplicated code like `PROFILE_NAME`. In this case, I think it would be preferable to use `UserFields` instead of `DarConstants`. `DarConstants` should eventually be deprecated altogether and `UserFields` seems like a more logical home for how we define the user's profile name. I think in an ideal world, we'd have a json schema to define these keys.